platform/linux-generic/odp_init.c

```diff
@@ -692,6 +692,26 @@ int odp_term_local(void)
 	return term_local(ALL_INIT);
 }
 
+int odp_term_abnormal(odp_instance_t instance, uint64_t flags, void *data ODP_UNUSED)
```

Comment:
Patch is otherwise OK, but git commit log entry should start with "linux-gen:" prefix.